config/database.yml

```diff
@@ -17,6 +17,9 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  host: localhost
+  username: postgres
+  password: alceconbotas2
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
```

Comment:
Do not upload this file to github (add it to the .gitignore), as it can generate conflicts with team members, or people who review your project. Instead I recommend using a `database.yml.example` file, where you put the general basic structure of that file and that people who download your project can use the .example to make their own database configuration file.